Alpha_shapes_2/doc/Alpha_shapes_2/CGAL/Alpha_shape_2.h

```diff
@@ -16,20 +16,34 @@ and the \f$ k\f$-dimensional faces of the triangulation.
 Note that this class is at the same time used for <I>basic</I> and 
 for <I>weighted</I> Alpha Shapes. 
 
+The modifying functions `Alpha_shape_2::insert()` and `Alpha_shape_2::remove()` will overwrite
+the one inherited from the underlying triangulation class `Dt`.
+At the moment, only the static version is implemented.
+
 \tparam Dt must be either `Delaunay_triangulation_2` or `Regular_triangulation_2`. 
-Note that `Dt::Geom_traits`, `Dt::Vertex` and `Dt::Face` must be model the concepts `AlphaShapeTraits_2`, 
+Note that `Dt::Geom_traits`, `Dt::Vertex`, and `Dt::Face` must be model the concepts `AlphaShapeTraits_2`,
 `AlphaShapeVertex_2` and `AlphaShapeFace_2`, respectively. 
 
 \tparam ExactAlphaComparisonTag is a tag that, when set to 
 \link Tag_true `Tag_true`\endlink, triggers exact comparisons between alpha values. This is useful 
 when the underlying triangulation is instantiated with an exact predicates inexact constructions 
 kernel. By default the `ExactAlphaComparisonTag` is set to \link Tag_false `Tag_false`\endlink as it induces a small 
-overhead. Note that since such a strategy does not make sense if used together with a traits class with exact constructions, 
-the tag `ExactAlphaComparisonTag` is not taken into account if `Dt::Geom_traits::FT` is not a floating point number type. 
-
-The modifying functions `Alpha_shape_2::insert()` and `Alpha_shape_2::remove()` will overwrite
-the one inherited from the underlying triangulation class `Dt`.
-At the moment, only the static version is implemented.
+overhead. Note that the tag `ExactAlphaComparisonTag` is currently ignored (meaning that the code will
+ behave as if `ExactAlphaComparisonTag` were set to \link Tag_false `Tag_false`\endlink)
+if `Dt::Geom_traits::FT` is not a floating point number type as this strategy
+does not make sense if the traits class already provides exact constructions.
+
+\warning When the tag `ExactAlphaComparisonTag` is set to \link Tag_true `Tag_true`\endlink,
+the class `Cartesian_converter` is used internally to switch between the traits class
+and exact CGAL kernels. `Cartesian_converter` must thus provide the necessary functor
```

Comment:
... and `CGAL::Simple_cartesian<NT>` where NT can be either `CGAL::Interval_nt` or `CGAL::Exact_rational`